packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -145,13 +156,39 @@ const resultsCount = computed(() => {
 	return total;
 });
 
+const optionBoolstoStrs = {
+	similarContent: 'similar content',
+	forwardCitation: 'forward citations',
+	backwardCitation: 'backward citations',
+	relatedContent: 'related resources'
+};
+
+const resultsStr = computed(() => {
+	let s = '';
+
+	if (searchByExampleAssetCardProp.value) {
+		Object.entries(searchByExampleOptions.value).forEach((item) => {
+			const [key, value] = item;
+			if (value) {
+				if (s.length > 0) {
+					s += ', ';
+				}
+				s += optionBoolstoStrs[key];
+			}
+		});
+	}
+
+	return s;
+});
+
```

Comment:
If you are making a string that contains the same value every time, why not just a string literal?